ansible_base/oauth2_provider/views/token.py

```diff
@@ -0,0 +1,57 @@
+from datetime import timedelta
+
+from django.utils.timezone import now
+from oauth2_provider import views as oauth_views
+from oauthlib import oauth2
+from rest_framework import permissions
+from rest_framework.viewsets import ModelViewSet
+
+from ansible_base.lib.utils.settings import get_setting
+from ansible_base.lib.utils.views.django_app_api import AnsibleBaseDjangoAppApiView
+from ansible_base.oauth2_provider.models import OAuth2AccessToken, OAuth2RefreshToken
+from ansible_base.oauth2_provider.serializers import OAuth2TokenSerializer
+
+
+class TokenView(oauth_views.TokenView, AnsibleBaseDjangoAppApiView):
+    # There is a big flow of logic that happens around this behind the scenes.
+    #
+    # oauth2_provider.views.TokenView inherits from oauth2_provider.views.mixins.OAuthLibMixin
+    # That's where this method comes from originally.
+    # Then *that* method ends up calling oauth2_provider.oauth2_backends.OAuthLibCore.create_token_response
+    # Then *that* method ends up (ultimately) calling oauthlib.oauth2.rfc6749....
```

Comment:
This apply to the create_token_response method or the class? Can we specify whatever it is instead of the term `this`?